stories/message-page/message-page.stories.js

```diff
@@ -0,0 +1,198 @@
+import '../../dist/message-page.css';
+
+export default {
+    title: 'Components/Message Page',
+    parameters: {
+        description: `A message page gives feedback to the user when an app or list is empty or if an error has occurred. The text and icon on the message page changes depending on the use case, however, the layout stays the same.
+        
+**The message page can be used in the following situations:**
+
+- Filtering with no results
+- Searching with no results
+- Empty app
+- Too many items
+- Item saved as a tile that is not longer available
+- Error
+
+## Responsiveness
+The size of the message page adjusts to fit the available space.
+
+## Guidelines
+
+- The icon in the message page is mandatory, and can vary depending on the message. The default icon is <code class="docs-code">.sap-icon--documents</code>.
+- The title is also mandatory. Add a description with further details if needed. 
+- The message should be as concise as possible.
+        `,
+        tags: ['f3', 'a11y', 'theme', 'development']
+    }
+};
+
+const messageBoxHeight = 800;
+
+export const filter = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--filter fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
```

Comment:
We could use a `role="status"` and `aria-live="polite"` on this message so the message is announced to the a screen reader user when it appears.

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reply:
Thanks @meganaconley I added the role and aria-live to `<div class="fd-message-page__content">`
Thank you for providing the links.